warp/verifier_backend.go

```diff
@@ -36,6 +38,8 @@ func (b *backend) Verify(ctx context.Context, unsignedMessage *avalancheWarp.Uns
 	}
 
 	switch p := parsed.(type) {
+	case *payload.AddressedCall:
```

Comment:
Not new in this PR, but should we check above on line 30 if `err != MessageNotFound`, and return the error in that case?

Reply:
yea good idea.